x-pack/plugins/security_solution/public/detections/components/user_info/index.test.tsx

```diff
@@ -38,29 +38,25 @@ describe('useUserInfo', () => {
     jest.spyOn(sourcererSelectors, 'signalIndexMappingOutdated').mockReturnValue(null);
   });
   it('returns default state', async () => {
-    await act(async () => {
-      const { result, waitForNextUpdate } = renderHook(() => useUserInfo(), {
-        wrapper: TestProviders,
-      });
-      await waitForNextUpdate();
+    const { result } = renderHook(() => useUserInfo(), {
+      wrapper: TestProviders,
+    });
+    await waitFor(() => new Promise((resolve) => resolve(null)));
 
-      expect(result.all).toHaveLength(1);
-      expect(result.current).toEqual({
-        canUserCRUD: null,
-        canUserREAD: null,
-        hasEncryptionKey: null,
-        hasIndexManage: null,
-        hasIndexMaintenance: null,
-        hasIndexWrite: null,
-        hasIndexRead: null,
-        hasIndexUpdateDelete: null,
-        isAuthenticated: null,
-        isSignalIndexExists: null,
-        loading: true,
-        signalIndexName: null,
-        signalIndexMappingOutdated: null,
-      });
-      expect(result.error).toBeUndefined();
+    expect(result.current).toEqual({
```

Comment:
why removing `expect(result.all).toHaveLength(1);` and `expect(result.error).toBeUndefined();` here?

Reply:
`renderHook` no longer returns the `result.all` and `result.error` property